modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -9073,6 +9124,16 @@ type TargetEnvironment {
     includeDeleted: Boolean! = false
   ): Target
 
+  """
+  The guide star(s) and related information
+  """
+  guideEnvironment(
+    """
+    The time used for the guide star calculations.
+    """
+    observationTime: Timestamp!
+  ): GuideEnvironment
```

Comment:
At some point in a future PR we'll need, I think, an option to specify a duration (defaulting to the entire planned time presumably). The duration will impact the average parallactic angle computation.

Reply:
Right. There is going to need to be some significant tinkering here. I'm not sure how average parallactic is being handled now without it. 🤔